linker/shared/src/test/scala/org/scalajs/linker/AnalyzerTest.scala

```diff
@@ -537,12 +576,13 @@ object AnalyzerTest {
 
   private def computeAnalysis(classDefs: Seq[ClassDef],
       symbolRequirements: SymbolRequirement = reqsFactory.none(),
+      config: CommonPhaseConfig = CommonPhaseConfig(),
```

Comment:
Consider taking a `StandardConfig` here. This hopefully makes the `config` helper above unnecessary.

Reply:
Good point. Done.